ci/hera_mc_test.yaml

```diff
@@ -0,0 +1,31 @@
+name: hera_mc_test
+channels:
+  - defaults
+  - conda-forge
+  - plotly
+dependencies:
+  - alembic
+  - astropy
+  - cartopy
+  - chart-studio
+  - coveralls
+  - flake8
+  - flake8-docstrings
```

Comment:
I know you _can_ get this from conda, but it feels weird to me to get some flake8 plugins from conda and some from pip.

Reply:
ok. I can change it but it doesn't seem weird to me. I get everything I can from conda and only use pip for what I can't get from conda.

Reply:
I used to be like this too, then i installed `black` through conda-forge and it was broken. I'm not dead set on the change, maybe just being grumpy